HH_custom_primary_key.md

```diff
@@ -0,0 +1,97 @@
+###Example: Create a model with custom primary key `name:string` instead of autogenerated `id:integer`
+
```

Comment:
What would you think about adding some intro text here? I think it would be good to give readers an idea of why we would choose to do this. What problem are we solving? Is this primarily for legacy systems or new?

It might be good to mention that while Ecto makes this possible, it's not necessarily the natural path Ecto wants us to take.